src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13012,6 +13034,7 @@ void VRFTerminalUnitEquipment::CalcVRF_FluidTCtrl(EnergyPlusData &state,
                       _,
                       _,
                       state.dataHVACVarRefFlow->MaxHeatingCapacity(VRFCond));
+            AirMassFlow = state.dataDXCoils->DXCoil(this->HeatCoilIndex).InletAirMassFlowRate;
```

Comment:
The variable AirMassFlow is not used until the LoadMet calculation below at new line 13108. I suggest moving this down to just after the supplemental heating coil call at new line 13078 using:

    AirMassFlow = state.dataLoopNodes->Node(VRFTUOutletNodeNum).MassFlowRate

Reply:
But the coil air flow is updated in `SimDXCoil`, should it be the updated coil air mass flow or TU air mass flow (probably need to be updated due to changes in coil air mass flow?) at new line 13078?

Reply:
No it is not. The coil inlet mass flow rate is passed to the outlet node. In the calc function there is a local variable that is updated to represent the maximum coil air flow rate and at the end of the calc function the outlet T/w/H are calculated based on the inlet mass flow rate, via the PLR variable.

Reply:
In InitDXCoil is this at line 7259:

    thisDXCoil.InletAirMassFlowRate = state.dataLoopNodes->Node(AirInletNode).MassFlowRate;

and in UpdateDXCoil is this at line 14193:

    state.dataLoopNodes->Node(AirOutletNode).MassFlowRate = thisDXCoil.InletAirMassFlowRate;

Reply:
Oh, I do see a place in CalcVRFHeatingCoil_FluidTCtrl that updates the inlet mass flow at line 17291:

        thisDXCoil.InletAirMassFlowRate = AirMassFlow * PartLoadRatio;

That does not look right. The TU should have placed a mass flow rate at the coil inlet.

Reply:
So when the child uses its local variable (fan speed ratio and its air flow, which might be different from the one passed in from the parent VRF TU) to calculate its own performance, then the coil heating rate and TU heating rate will be different due to the air flow rate difference? But for a simple case of TU containing a heating coil and a supply fan with no outdoor air load, shouldn't the TU and coil heating rate be roughly equal?

Reply:
Somehow the CalVRFTUAirFlowRate_FluidTCtrl and SimDXCoil have different results for the fan speed ratio. Maybe I need to resolve this difference. Currently the former almost always sets the air flow rate at the maximum value while the coil thinks it's smaller.

Reply:
I found the issue. The CalVRFTUAirFlowRate_FluidTCtrl doesn't have the MaxHeatCap limit like the ones in the SimDXCoil does.